src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
```

Comment:
```suggestion
import java.util.List;
```
Se corrige la importacion